src/components/list-facet/index.js

```diff
@@ -99,17 +99,35 @@ class FederatedListFacet extends React.Component {
       value,
       collapse,
       hierarchy,
+      options,
     } = this.props;
     const { truncateFacetListsAt } = this.state;
 
+    const siteList = options.siteList;
     const facetCounts = facets.filter((facet, i) => i % 2 === 1);
     const facetValues = facets.filter((facet, i) => i % 2 === 0);
     // Create an object of facets {value: count} to keep consistent for inputs.
     const facetInputs = {};
-    facetValues.forEach((v, i) => {
-      const key = facetValues[i];
-      facetInputs[key] = facetCounts[i];
-    });
+
+    // Handle site name restrictions.
+    if (field === 'sm_site_name' && siteList.length > 0) {
+      facetValues.forEach((v, i) => {
+        const key = facetValues[i];
+        if (siteList.indexOf(v) > -1) {
```

Comment:
I was trying to think of a way to avoid the if statement inside of the loop but I think filtering down the `facetValues` array based on `siteList` would change the index/key of the `facetValues`, so I think this is probably best.